.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -285,7 +294,9 @@ function parseJsonFromText(text) {
     // Parse the found string to JSON
     return JSON.parse(match[1]);
   } else {
-    throw new Error("JSON data not found");
+    // Temporary fix to bypass issue introduced by https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/c7ae39b72de203bbae0cea031728ed630f1dc293
```

Comment:
Do we have an issue number for this? I think we should fix this first if possible instead of introducing a workaround, as I understand it's entirely under our control

Reply:
@crobert-1 any thoughts on how we should tackle this?

Reply:
I'd prefer addressing separately, with a new issue opened for this. @LucaLanziani would you be able to open an issue?

Reply:
Issue created https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33388

Reply:
My suggestion would be to still merge this to fix the job while making the decision on issue #33388

The [JSON part](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/workflows/scripts/generate-weekly-report.js#L298-L305) (to be more precise the previous issue presence) was always optional and the code support that not being present, I would even "question" the initial idea of throwing the error given the optionality of the functionality.

Reply:
oh sure, if the decision is already made in the issue (as I can see it is) happy to split the commits